cips/cip-X.md

```diff
@@ -262,9 +262,6 @@ Backwards compatibility is guaranteed as this CIP introduces new messages but do
 
 ## Security Considerations
 
-It is important to notice that the `Data` field in `MsgSignArbitraryData` consists of bytes, which means it could be
-non-human-readable.
```

Comment:
I think this can be rephrased but can remain here. Valid UTF-8 string does not mean
1. the string is printable
2. the signing device is capable of printing it

Examples of such valid strings are NULL, [BELL](https://en.wikipedia.org/wiki/Bell_character), emojis or unicode in the [Private Use Areas](https://en.wikipedia.org/wiki/Private_Use_Areas).